pytest_html/plugin.py

```diff
@@ -113,8 +113,6 @@ def __init__(self, outcome, report, logfile, config):
             self.config = config
             self.row_table = self.row_extra = None
 
-            test_index = hasattr(report, 'rerun') and report.rerun + 1 or 0
```

Comment:
I realised from [here](https://github.com/pytest-dev/pytest-rerunfailures/commit/4ec74311cd5b199a8ec6f198f17ea77ea16c56d6#diff-c3a581865b7c0be4e6b5d5fa60d6aed5) that report doesn't have the attribute `rerun` anymore, so that was causing some `rerun` tests to fail. Instead of getting the test_index from the attribute, I'm passing it as a parameter from the property `rerun` of the `TestResult` class.

Reply:
This could be a regression from pytest-rerunfailures. It would seem that the latest version doesn't report the number of reruns in the report? If so, please raise an issue against the plugin. In the meantime we could pin our tests to the version of pytest-rerunfailures before this change.